### DISCUSSION TITLE
Workspace shorthand name

### DISCUSSION THREAD

**Discussion by joezappie** [npm/feedback#708] [CLI] [answered]
I think it would be great if you could define a shorthand workspace name in the package.json. If I want to give my package a descriptive name like @company/project that becomes tedious to type in the NPM cli. For example:

```
npm i express -w @mycompany/project-a
```

What about having a "workspaceName" or "shortName" property that lets you define a shorter name for internal use:

```
{
    "name": "@mycompany/project-a",
    "workspaceName": "pa",
}

npm i express -w pa
```

While I don't personally have any super long package names yet, I have seen some projects that have 3-4 word names or have words that are just hard to spell. I definitely would get use out of a the shorthand name ability though.

Just started playing with workspaces but from my googling I didn't find any info that this already exists.

**Comment by MylesBorins** [comment, marked as answer]
The next step for a feature proposal like this would be to open an issue over at https://github.com/npm/rfcs

That's for the suggestion!
